low-liquidity-market-attack-monitor/src/agent.spec.js

```diff
@@ -0,0 +1,380 @@
+const mockedGetContract = jest.fn();
```

Comment:
These tests need to be updated to work with the newer version of the Forta SDK.

I ran this locally and the tests that used the `mockTxEvent` Object failed:
```bash
rm -rf node_modules package-lock.json
npm i
npm test
```

The arguments for the `TransactionEvent` constructor have changed: https://github.com/forta-protocol/forta-agent-sdk/blob/d46e9a7e1f547296cc85c23fae7c22e8c3ce3fbf/sdk/transaction.event.ts#L19

Here is an example of us using the newer argument ordering:
https://github.com/arbitraryexecution/forta-agent-templates/blob/be74f81f86f5dbceaa80e4b4a155ed1f3bde1a31/gnosis-safe-multisig/src/agent.spec.js#L189

The `receipt` key no longer exists, but the `logs` key has been promoted to be directly under the `txEvent` Object.